main.js

```diff
@@ -1,6 +1,8 @@
+'use strict';
+
 // Dependencies
 const { token } = require('./config.json');
-const bot = require('./src/bot.js');
```

Comment:
No need to specify `.js` in require expression.
Weirdly enough this is the only place I found, so it's likely that you forgot to remove it.

Reply:
Yeah i forgot to remove it